README.adoc

```diff
@@ -69,78 +74,52 @@ include::{common-includes}/gitclone.adoc[]
+The [hotspot=systemLoadMethod file=1]`systemLoad` endpoint asynchronously processes the data that is retrieved by the [hotspot=getsystem file=1]`InventoryClient` and serves the data when all of the services have responded. The [hotspot=thenAcceptAsync file=1]`thenAcceptAsync()` and [hotspot=exceptionally file=1]`exceptionally()` methods together behave like an asynchronous try-catch block. The data is processed in the [hotspot=thenAcceptAsync file=1]`thenAcceptAsync()` method only when the `CompletionStage` interface has finished retrieving it. When you return a `CompletionStage` type in the resource, it doesn’t necessarily mean that the computation completed and the response was built.
 
-Instead of using `thenAcceptAsync()`, `Observable` uses the [hotspot=subscribe file=0]`subscribe()` method to asynchronously process data. Thus, any required processing of data will be performed in the [hotspot=subscribe file=0]`subscribe()` method. In this case, it is simply saving the data in the temporary [hotspot=holder hotspot=holderClass file=0]`holder`. The temporary [hotspot=holder hotspot=holderClass file=0]`holder` is used to store the value returned from the client because values cannot be returned inside [hotspot=subscribe file=0]`subscribe()`. The highest and lowest load systems are updated in the [hotspot=updateHighestCall hotspot=updateHighestMethod file=0]`updateHighest()` and [hotspot=updateLowestCall hotspot=updateLowestMethod file=0]`updateLowest()` methods respectively.
-
-A [hotspot=countdownlatch file=0]`CountDownLatch` is used to keep track of how many asynchronous requests are being waited on. The [hotspot=countdownlatch file=0]`CountDownLatch` will countdown towards 0 after the completion of a thread via the [hotspot=countdown hotspot=countdown2 file=0]`countdown` method call. This means that the value will only return once the thread that's retrieving the value is complete. The [hotspot=await file=0]`await` method stops and waits until all of the requests have completed. While waiting for this countdown to complete, the main thread is free to perform other tasks. In this case, no such task is present, but a simple task can be given to occupy the thread.
-
-// =================================================================================================
-// Building the application
-// =================================================================================================
+A [hotspot=countdownlatch file=1]`CountDownLatch` object is used to keep track of how many asynchronous requests are being waited on. 
+The `CountDownLatch` object will countdown towards 0 after each completion of a thread via the [hotspot=countdown1 hotspot=countdown2 file=1]`countdown` method call. 
```

Comment:
Make this present tense: "...object counts down towards `0` after..." It's describing how the technology works so always present tense.